courseSchedule.py

```diff
@@ -0,0 +1,36 @@
+"""
+Time complexity: O(N)
+Space complexity: O(N)
+Compiled on leetcode: Yes
```

Comment:
time complexity is V*E and space complexity is V+E where V is vertices and E is Edges